src/js/view-step3/inputLottoNumsView.js

```diff
@@ -0,0 +1,64 @@
+import { $ } from '../utils/dom';
+
+export class InputLottoNumsView {
+  #inputLottoNums;
+  #showResultBtn;
+  #inputs;
+  #inputValues = [];
+  #first;
+  #second;
+  #third;
+  #fourth;
+  #fifth;
+  #rateOfReturn;
+
+  constructor() {
+    this.#inputLottoNums = $('.input-lotto-nums');
+    this.#showResultBtn = $('.show-result-btn');
+    this.#inputs = this.#inputLottoNums.querySelectorAll('input');
+    this.#first = $('.first');
+    this.#second = $('.second');
+    this.#third = $('.third');
+    this.#fourth = $('.fourth');
+    this.#fifth = $('.fifth');
+    this.#rateOfReturn = $('.rate-of-return');
+
+    this.#inputLottoNums.style.display = 'none';
+  }
+
+  getWinningAndBonusNumbers() {
+    this.#inputs.forEach((input) => this.#inputValues.push(input.value));
```

Comment:
특별한 의도가 있는 것이 아니라면 가능한 선언적으로 작성해보시면 좋을 것 같습니다!

```suggestion
    this.#inputValues = this.#inputs.map(input => input.value)
```